tests/validation/tests/v3_api/test_v2_cis_scan.py

```diff
@@ -68,10 +78,203 @@ def test_install_v2_cis_benchmark():
+        cis_scan_report_results[benchmark_version]["skip"])
+
+
+def test_v2_create_clusterscanprofile():
+    """
+    Create a custom Scan profile
+    Run scan using this profile
+    Validate the scan CR and the scan report generated
+    """
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    rancherrepo = \
+        client.list_catalog_cattle_io_clusterrepo(id="rancher-charts")
+    cluster_id = cluster_detail["cluster"]["id"]
+    cluster_name = cluster_detail["cluster"]["name"]
+    rancher_repo = rancherrepo["data"][0]
+
+    # check if CIS is already installed and uninstall the app
+    if not check_v2_app_installed(client,CHART_NAME):
+        install_cis_v2(client, cluster_id, cluster_name, rancher_repo)
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    # create clusterscanprofile
+    benchmark_version = "rke-cis-1.5-permissive"
+    tests_to_skip = ["1.1.11", "1.1.19"]
+    # these tests as in "pass" state in an rke permissive scan run
+    profile = create_clusterscanprofile(
+        client, benchmark_version, tests_to_skip
+    )
+    print("profile", profile)
+    assert profile["metadata"]["state"]["name"] == "active"
+    assert profile["spec"]["skipTests"] == tests_to_skip
+    assert profile["spec"]["benchmarkVersion"] == benchmark_version
+
+    # create a clusterscan using this profile
+    scan = create_clusterscan(client, profile["id"])
+
+    # validate scan CR
+    tests_skipped = len(tests_to_skip)
+    validate_clusterscan_cr_and_report(
+        client, scan, profile["id"], benchmark_version,
+        cis_scan_report_results[benchmark_version]["total"],
+        cis_scan_report_results[benchmark_version]["pass"] - tests_skipped,
+        cis_scan_report_results[benchmark_version]["fail"],
+        cis_scan_report_results[benchmark_version]["notApplicable"],
+        cis_scan_report_results[benchmark_version]["skip"] + tests_skipped
+    )
+
+
+def test_v2_create_clusterscanbenchmark():
+    """
+    This test will fail because of issue : 
+    https://github.com/rancher/cis-operator/issues/44
+    Create a benchmark version similar to "rke-cis-1.5-permissive"
+    Create a cluster scan profile using this benchmark
+    Create a cluster scan using this profile
+
+    """
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"], USER_TOKEN
+        )
+    rancherrepo = \
+        client.list_catalog_cattle_io_clusterrepo(id="rancher-charts")
+    cluster_id = cluster_detail["cluster"]["id"]
+    cluster_name = cluster_detail["cluster"]["name"]
+    rancher_repo = rancherrepo["data"][0]
+
+    # check if CIS is already installed and uninstall the app
+    if not check_v2_app_installed(client,CHART_NAME):
+        install_cis_v2(client, cluster_id, cluster_name, rancher_repo)
```

Comment:
The comment message is misleading: do you uninstall the app here? 
(this comment appears at multiple places)

Reply:
Sorry about that. Will change it.